src/cf.data.pre

```diff
@@ -2418,6 +2418,30 @@ DOC_START
 			The proxy_protocol_access is required to whitelist
 			downstream proxies which can be trusted.
 
+	   worker-queues
+			Ask TCP stack to maintain a dedicated listening queue
+			for each worker accepting requests at this port.
+			Requires TCP stack that supports the SO_REUSEPORT socket
+			option.
+
+			Cons: Enabling worker-specific queues allows any process
```

Comment:
It is a bit silly to warn admins that users with same-as-Squid rights can spy on or hurt Squid. With or without worker-queues, a user that can start processes as the Squid user can accept requests destined to a Squid port. It is just _easier_ to do so when worker-queues are enabled.

Said that, I do not object to `SECURITY WARNING` in hope to save time if Amos insists on this addition. Just please split this paragraph in two (each with a `Cons:` prefix) so that the shouting does not apply to the second Cons.

@yadij, do you insist on this change?

Reply:
I do. The issue is not just spying. This greatly simplifies the actions needed to get ability to manipulate traffic an admin thinks is going to the proxy.

Reply:
@eduard-bagdasaryan, when you address this change request, please split this paragraph in two (each with a Cons: prefix) so that the shouting does not apply to the second Cons. I would also remove the exclamation point at the end of the first Cons -- the all-caps prefix alone is more than enough to bring the reader's attention -- but that is your call.

Reply:
Actually, the first paragraph after the split does not need a `Cons:` prefix at all! The all-caps prefix is enough.

Reply:
SECURITY WARNING was added, both Cons and Pros were removed, resolving another request.